### ISSUE TITLE
训练集标签

### ISSUE THREAD

**Issue by 1HykhqV3rU** [vision4robotics/UDAT#16]
请问训练集没有groundtruth吗，我下载的数据集只包含图片

**Comment by Jay-Ye** [comment]
Hello, thank you for your interest in our work. 
NAT2021 is designed for unsupervised domain adaptation and does not have ground truth labels for the training set.
However, you can follow the [preprocessing step](https://github.com/vision4robotics/UDAT#1-preprocessing) to generate pseudo labels.
You can also download the [test set](https://drive.google.com/drive/folders/11kwBWE7DVQ1InbK_yKnIVNgLu1u7v27B?usp=drive_link), which does provide gt labels.
